launch_testing/launch_testing/__init__.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         self.name = name
         self.launch_descriptor = launch_descriptor
         self.expected_lines = expected_lines
+        self.expected_output = b'\n'.join(self.expected_lines)
```

Comment:
How would you write your regex for the example:

    foo 1
    bar 2

if another line might be printed before?